functions/EmuScripts/emuDeckBigPEmu.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+#variables
+BigPEmu_emuName="BigPEmu (proton)"
+BigPEmu_emuType="windows"
+BigPEmu_emuPath="${romsPath}/Applications/BigPEmu/BigPEmu.exe"
+BigPEmu_appID=
+BigPEmu_appData="${HOME}/.steam/steam/steamapps/compatdata/${BigPEmu_appID}/pfx/drive_c/users/steamuser/AppData/Roaming/BigPEmu"
+BigPEmu_BigPEmuSettings="${BigPEmu_appData}/BigPEmuConfig.bigpcfg"
+
+#cleanupOlderThings
+BigPEmu_cleanup(){
+	echo "NYI"
+}
+
+#Install
+BigPEmu_install(){
+	setMSG "Installing $BigPEmu_emuName"
+
+	local showProgress="$1"
+    if wget -m -nd -A "BigPEmu_*.zip" -O "$HOME/Applications/BigPEmu.zip" "https://www.richwhitehouse.com/jaguar/index.php?content=download"; then
+		mkdir -p "$HOME/Applications/BigPEmu"
+		unzip -o "$HOME/Applications/BigPEmu.zip" -d "$HOME/Applications/BigPEmu"
+		rm -f "$HOME/Applications/BigPEmu.zip"
+	else
+		return 1
+	fi
+
+	cp "$EMUDECKGIT/tools/launchers/bigpemu.sh" "${toolsPath}/launchers/bigpemu.sh"
+
+	chmod +x "${toolsPath}/launchers/bigpemu.sh"
+
+	createDesktopShortcut   "$HOME/.local/share/applications/BigPEmu (Proton).desktop" \
+							"BigPEmu (Proton)" \
+							"${toolsPath}/launchers/bigpemu.sh -w"  \
+							"False"
+}
+
+#ApplyInitialSettings
+BigPEmu_init(){
+	setMSG "Initializing $BigPEmu_emuName settings."	
```

Comment:
this function needs to update es-de's custom systems xml file. i'd suggest you base it off xenia.